src/components/Header/TopNav/outerItemStates.js

```diff
@@ -17,12 +17,14 @@ const clickTapLight = css`
 
 const activeLight = css`
   background: ${props => props.theme.colors.white};
-  color: ${props => props.theme.colors.textLight};
+  color: ${props => props.theme.colors.text};
+  font-weight: bold;
 `
 
 const activeAndHoverLight = css`
-  background: ${props => props.theme.colors.text};
-  color: #a9a9a9;
+  background: ${props => props.theme.colors.greyBG};
+  color: ${props => props.theme.colors.text};
+  font-weight: bold;
```

Comment:
IMO, the bold is unnecessary and it really clashes with the rest of the topnav elements :(